manual/src/Makefile

```diff
@@ -1,10 +1,7 @@
-SRC = $(abspath ../..)
--include $(SRC)/Makefile.config
+ROOTDIR = ../..
+-include $(ROOTDIR)/Makefile.config
```

Comment:
Note that this include should be `Makefile.build_config` since its aim is to import  the odoc/ocamldoc configuration information (aka `DOCUMENTATION_TOOL`).